libvast/src/configuration.cpp

```diff
@@ -342,11 +360,17 @@ caf::error configuration::parse(int argc, char** argv) {
   // the function config_dirs() relies on this already being there.
   if (auto it
       = std::find(command_line.begin(), command_line.end(), "--bare-mode");
-      it != command_line.end())
+      it != command_line.end()) {
     caf::put(content, "vast.bare-mode", true);
-  else if (auto vast_bare_mode = detail::getenv("VAST_BARE_MODE"))
-    if (*vast_bare_mode == "true")
+  } else if (auto vast_bare_mode = detail::getenv("TENZIR_BARE_MODE")) {
+    if (*vast_bare_mode == "true") {
+      caf::put(content, "vast.bare-mode", true);
```

Comment:
Do you plan to adapt the `vast.` prefix later on?

Reply:
Yes, but that's a much bigger PR and more complex than this one.